.gitignore

```diff
@@ -1,137 +1,8 @@
-
-.svn/text-base/screenshot-1.png.svn-base
-
-.svn/text-base/readme.txt.svn-base
-
-.svn/text-base/screenshot-4.png.svn-base
-
-.svn/text-base/custom_metadata.php.svn-base
-
-css/.svn/entries
-
-css/.svn/all-wcprops
-
-css/.svn/text-base/custom-metadata-manager.css.svn-base
-
-css/.svn/prop-base/jquery-ui-smoothness.css.svn-base
-
-css/.svn/text-base/jquery-ui-smoothness.css.svn-base
-
-css/images/.svn/prop-base/ui-bg_flat_0_aaaaaa_40x100.png.svn-base
-
-css/images/.svn/entries
-
-css/images/.svn/all-wcprops
-
-css/images/.svn/prop-base/ui-bg_flat_75_ffffff_40x100.png.svn-base
-
-css/images/.svn/prop-base/ui-bg_glass_65_ffffff_1x400.png.svn-base
-
-css/images/.svn/prop-base/ui-bg_glass_65_ffffff_1x400.png.svn-base
-
-css/images/.svn/prop-base/ui-bg_glass_75_dadada_1x400.png.svn-base
-
-js/images/.svn/text-base/calendar.png.svn-base
-
-js/images/.svn/entries
-
-js/images/.svn/all-wcprops
-
-js/images/.svn/prop-base/calendar.png.svn-base
-
-js/.svn/text-base/jquery-ui-datepicker.min.js.svn-base
-
-js/.svn/text-base/custom-metadata-manager.js.svn-base
-
-js/.svn/prop-base/jquery-ui-datepicker.min.js.svn-base
-
-js/.svn/entries
-
-js/.svn/all-wcprops
-
-css/images/.svn/text-base/ui-icons_cd0a0a_256x240.png.svn-base
-
-css/images/.svn/text-base/ui-icons_454545_256x240.png.svn-base
-
-css/images/.svn/text-base/ui-icons_222222_256x240.png.svn-base
-
-css/images/.svn/text-base/ui-icons_2e83ff_256x240.png.svn-base
-
-css/images/.svn/text-base/ui-icons_2e83ff_256x240.png.svn-base
-
-css/images/.svn/text-base/ui-bg_highlight-soft_75_cccccc_1x100.png.svn-base
-
-css/images/.svn/text-base/ui-bg_glass_65_ffffff_1x400.png.svn-base
-
-css/images/.svn/text-base/ui-icons_888888_256x240.png.svn-base
-
-css/images/.svn/text-base/ui-bg_glass_75_e6e6e6_1x400.png.svn-base
-
-css/images/.svn/text-base/ui-bg_flat_75_ffffff_40x100.png.svn-base
-
-css/images/.svn/prop-base/ui-icons_888888_256x240.png.svn-base
-
-css/images/.svn/text-base/ui-bg_flat_0_aaaaaa_40x100.png.svn-base
-
-css/images/.svn/prop-base/ui-icons_cd0a0a_256x240.png.svn-base
-
-css/images/.svn/prop-base/ui-icons_454545_256x240.png.svn-base
-
-css/images/.svn/prop-base/ui-icons_2e83ff_256x240.png.svn-base
-
-css/images/.svn/text-base/ui-bg_glass_55_fbf9ee_1x400.png.svn-base
-
-css/images/.svn/prop-base/ui-icons_222222_256x240.png.svn-base
-
-css/images/.svn/prop-base/ui-bg_glass_95_fef1ec_1x400.png.svn-base
-
-css/images/.svn/prop-base/ui-bg_highlight-soft_75_cccccc_1x100.png.svn-base
-
-css/images/.svn/prop-base/ui-bg_highlight-soft_75_cccccc_1x100.png.svn-base
-
-css/images/.svn/prop-base/ui-bg_glass_75_e6e6e6_1x400.png.svn-base
-
-css/images/.svn/text-base/ui-bg_glass_75_dadada_1x400.png.svn-base
-
-css/images/.svn/text-base/ui-bg_glass_95_fef1ec_1x400.png.svn-base
-
-css/images/.svn/prop-base/ui-bg_glass_55_fbf9ee_1x400.png.svn-base
-
-.svn/lock
-
-.svn/dir-props
-
-.svn/tmp/text-base/readme.txt.svn-base
-
-.svn/dir-prop-base
-
-.DS_store
-.DS_Store
-js/.DS_Store
-css/.DS_Store
 .svn/
```

Comment:
None of the others legacy plugins ignore `.svn/`, and once we have the deploy.yml in place, then SVN doesn't need to be touched again.